src/sentry/search/events/types.py

```diff
@@ -55,14 +56,18 @@ class QueryFramework:
     entity: Entity
 
 
+SnubaRow = dict[str, Any]
```

Comment:
afaik this is the best we can do for Snuba's response, we know the keys are strings matching our field names, but the values can literally be anything

Reply:
Tried typing this with:
`JSON: TypeAlias = dict[str, "JSON"] | list["JSON"] | str | int | float | bool | None`
Which then made everything using results break since we aren't typing the usage of SnubaResults much yet. Going to leave this as Any for now